weave-js/src/components/Panel2/PanelTable/ColumnHeader.tsx

```diff
@@ -122,6 +125,8 @@ export const ColumnHeader: React.FC<{
   panelContext: any;
   isPinned: boolean;
   simpleTable?: boolean;
+  countColumnId?: string | null;
```

Comment:
Do `countColumnId` and `setCountColumnId` need to be optional? Looks like we always pass them.

Reply:
made them required